trace/HTTP.md

```diff
@@ -29,24 +29,28 @@ incoming requests should be a span kind of SERVER.
 
 Propagation is how SpanContext is transmitted on the wire in an HTTP request.
 
-Implementations MUST allow users to set their own propagation format and MUST provide an 
+Implementations MUST allow users to set their own propagation format and MUST provide an
 implementation for B3 at least.
 
+Implementations should provide a way to avoid propagating SpanContext altogether from both
+inbound requests and on outbound requests while still generating spans as usual
+around the inbound/outbound request.
```

Comment:
This section should have a use case as this implies complexity. For example, you'd need some sort of functionality to express which endpoints are those which you'd not want to propagate downstream to. Also we are moving features here which could be addressed individually with middleware (header scrubbing for various reasons can be site specific)